Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts

```diff
@@ -6,6 +6,7 @@ type Notification = {
     id: string,
     isCompleted: boolean,
     sentDate: string,
+    sendingStartedDateTime: string,
```

Comment:
Rename to "sendingStartedDate" because I'm having the model's name changed.

Reply:
updated